webapp/certified/views.py

```diff
@@ -102,12 +101,12 @@ def _parse_query_params(all_releases, all_vendors):
             "servers",
             "iot",
             "socs",
```

Comment:
If you add a new `Server SoC` category, aren't existing ones (`socs` and `SoC`) become redundant?

Reply:
No, because the `socs` are coming from our JS, and the `Server SoC` are from the Python code because the API only accepts `Server SoC`, it doesn't understand what is `socs`. So this is something I missed in the security PR.